backend/models/criterion.go

```diff
@@ -0,0 +1,24 @@
+package models
+
+import (
+	"backend/database"
+	"context"
+
+	"go.mongodb.org/mongo-driver/bson"
+	"go.mongodb.org/mongo-driver/bson/primitive"
+)
+
+type Criterion struct {
+	ID        primitive.ObjectID `json:"id"`
+	Dimension string             `json:"dimension"`
+	Criterion string             `json:"criterion"`
+}
+
+func GetRatingCriterionFromId(id primitive.ObjectID) (*Criterion, error) {
+	collection := database.GetInstance().Database("RateMyPeersDB").Collection("Criteria")
+	var rc Criterion
```

Comment:
i assume rc stands for "ratingCriteria" right?

Reply:
Oh you're right it should've been just `c`